elastic/kibana
# [App Search] New Precision Slider for Relevance Tuning

## You need to be running the ~`byronhulcher/update-precision`~ `master` branch in `elastic/ent-search` for this PR to work

## QA
1. Create an engine
2. Add documents
3. Go to Relevance Tuning for that engine
- [x] There should be a Precision slider
4. Change the value of this slider
- [x] It should update the slider
- [x] The description should change with each step
5. Save your updates
6. Refresh the page
- [x] The value you set should still be present
